packages/shorebird_cli/lib/src/commands/patch/ios_patcher.dart

```diff
@@ -97,12 +116,20 @@ class IosPatcher extends Patcher {
 
       if ((flutterVersion ?? minimumSupportedIosFlutterVersion) <
           minimumSupportedIosFlutterVersion) {
-        logger.err(
-          '''
+        final overrideMinFlutterVersion =
+            argResults[CommonArguments.overrideMinFlutterVersionArg.name] ==
+                true;
+        if (!overrideMinFlutterVersion) {
+          logger.err(
+            '''
 iOS patches are not supported with Flutter versions older than $minimumSupportedIosFlutterVersion.
-For more information see: $supportedVersionsLink''',
-        );
-        exit(ExitCode.software.code);
+For more information see: $supportedVersionsLink
+
+If you are certain about the compatibility of your Flutter version, you can bypass this check by passing the --override-min-flutter-version flag. Patches generated with the override flag must be sent to staging first, so make sure to also pass the --staging flag alongside.
```

Comment:
I don’t think anyone can be certain given how these issues happen at the linker and compiler level 🙁